quickstart/201-k8s-cluster-with-tf-and-aks/sp.tf

```diff
@@ -0,0 +1,28 @@
+# Create Azure AD App Registration
+resource "azuread_application" "app" {
+  display_name = "my-app"
+  owners       = [local.current_user_id]
+}
+
+# Create Service Principal
+resource "azuread_service_principal" "app" {
+  application_id               = azuread_application.app.application_id
+  app_role_assignment_required = true
+  owners                       = [local.current_user_id]
+}
+
+# Create Service Principal password
+resource "azuread_service_principal_password" "app" {
+  service_principal_id = azuread_service_principal.app.id
+}
+
+# Output the Service Principal and password
+output "sp" {
```

Comment:
Could we move these `output` blocks to `outputs.tf` file?

Reply:
@lonegunmanb Would it be possible to have the SP.tf and SSH.tf files fully encapsulated so that they stand alone and can be copied between projects more easily?